pcdet/datasets/nuscenes/nuscenes_dataset.py

```diff
@@ -137,6 +181,60 @@ def __getitem__(self, index):
                 'gt_names': info['gt_names'] if mask is None else info['gt_names'][mask],
                 'gt_boxes': info['gt_boxes'] if mask is None else info['gt_boxes'][mask]
             })
+        if self.use_camera:
```

Comment:
It will be better to add a new function to include all following added codes. I think this will make `__getitem__` easier to understand.

Reply:
I will define a function called `load_camera_info` to include these codes.

Reply:
Get it. Thanks a lot for your contribution. I think after the following adjustment, we can merge this PR. I will make sure with shaoshuai @sshaoshuai.